### PR TITLE
T240886 app package

### PR DESCRIPTION
**Pull request by stephanebisson**
Phabricator Link: https://phabricator.wikimedia.org/T240886

### Problem Statement

This application needs to be bundled in a zip to be distributed to stores. Also, the manifest needs to be updated with application info in various languages.

### Solution

`scripts/manifest.js` generates a new manifest with updated locales and version

`scripts/zip.sh` copies all needed files and folders to a temp dir and makes a zip file called app.zip

The script above is executed in CI and the resulting bundle is saved as an artifact in CircleCI.

### CODE REVIEW

**Comment by medied** [comment]
So I ran the script to test this PR, here's what the console output looked like for me (this was up to commit 74ef6f25b23b1dca6a9fa0ff39dc593f5ddd80ca):

```
emedina@Eduardos-MacBook-Pro wikipedia-kaios % sh scripts/zip.sh
npm ERR! missing script: Wikipedia


npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/emedina/.npm/_logs/2020-03-18T20_26_48_243Z-debug.log
cp: dist: No such file or directory
  adding: Wikipedia/ (stored 0%)
  adding: Wikipedia/index.html (deflated 37%)
  adding: Wikipedia/images/ (stored 0%)
  adding: Wikipedia/images/icon-gallery.svg (deflated 37%)
  adding: Wikipedia/images/icon-audio.svg (deflated 40%)
  adding: Wikipedia/images/icon-languages.svg (deflated 43%)
  adding: Wikipedia/images/onboarding-2-background.png (deflated 13%)
  adding: Wikipedia/images/loading-galaxy.svg (deflated 22%)
  adding: Wikipedia/images/112px.png (deflated 0%)
  adding: Wikipedia/images/84px.png (stored 0%)
  adding: Wikipedia/images/onboarding-2.png (stored 0%)
  adding: Wikipedia/images/onboarding-3.png (deflated 0%)
  adding: Wikipedia/images/onboarding-1.png (deflated 0%)
  adding: Wikipedia/images/onboarding-3-background.png (deflated 13%)
  adding: Wikipedia/images/loading-earth.svg (deflated 48%)
  adding: Wikipedia/images/onboarding-0.png (deflated 0%)
  adding: Wikipedia/images/onboarding-1-background.png (deflated 16%)
  adding: Wikipedia/images/link.svg (deflated 38%)
  adding: Wikipedia/images/loading-moon.svg (deflated 54%)
  adding: Wikipedia/images/article-error.png (stored 0%)
  adding: Wikipedia/images/wikipedia-wordmark-en.png (deflated 67%)
  adding: Wikipedia/images/56px.png (stored 0%)
  adding: Wikipedia/images/w.svg (deflated 54%)
  adding: Wikipedia/images/icon-quickfacts.svg (deflated 43%)
  adding: Wikipedia/images/search-offline.svg (deflated 84%)
  adding: Wikipedia/images/icon-down-arrow.svg (deflated 45%)
  adding: Wikipedia/images/icon-sections.svg (deflated 37%)
  adding: Wikipedia/manifest.webapp (deflated 64%)
emedina@Eduardos-MacBook-Pro wikipedia-kaios %
emedina@Eduardos-MacBook-Pro wikipedia-kaios %
emedina@Eduardos-MacBook-Pro wikipedia-kaios % ls
LICENSE                manifest.webapp
README.md            node_modules
Wikipedia            package-lock.json
app.zip                package.json
cypress                pull_request_template.md
cypress.json            scripts
i18n                src
images                style
index.html            webpack.config.js
emedina@Eduardos-MacBook-Pro wikipedia-kaios % ls Wikipedia
images        index.html    manifest.webapp
```

Indeed `app.zip` is there and although I'm not exactly sure what `manifest.webapp` is supposed to look like, it seems valid

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ cypress/screenshots
 cypress/videos/
 cypress/fixtures
 .vscode
+Wikipedia
+app.zip
```
Comment (medied) [inline .gitignore]: Ah yea good idea

File: scripts/zip.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+rm -rf Wikipedia
+mkdir Wikipedia
+
+rm -rf dist
+
+node scripts/manifest.js > Wikipedia/manifest.webapp
+npm run Wikipedia
```
Comment (hueitan) [inline scripts/zip.sh]: Is this `npm run build`, or something else?

Comment (stephanebisson) [reply scripts/zip.sh]: Yes, I broke it when I renamed the directory from "build" to "Wikipedia". Should be fixed now.

**Comment by hueitan** [comment]
Looks good to me
===

Except:

I got this error on Windows `stdout is not a tty`

https://superuser.com/questions/1011597/what-does-an-output-is-not-a-tty-error-mean/1350903

should we fix?

**Comment by stephanebisson** [comment]
> I got this error on Windows `stdout is not a tty`

Windows is not our target OS in any scenario (running or building) so... no.

**Pull request merged**